sips/sip-034/sip-034.md

```diff
@@ -0,0 +1,138 @@
+# Preamble
+
+SIP Number: 034
+
+Title: Dimension-Specific Tenure Extend Variants 
+
+Author(s):
+
+- Aaron Blankstein <aaron@stackslabs.com>
+- Jude Nelson <jude@stacks.org>
+
+Status: Draft
+
+Consideration: Technical, Governance
+
+Type: Consensus
+
+Layer: Consensus (hard fork)
+
+Created: 2025-10-15
+
+License: CC0-1.0
+
+# Abstract
+
+This SIP details a rider to SIP-033, including a new variant of the tenure change
+control transaction. This variant allows the tenure change to specify a specific 
+dimension of the block budget to reset.
+
+# Copyright
+
+This SIP is made available under the terms of the CC0 (Creative Commons Zero)
+license, available at https://opensource.org/licenses/CC0-1.0. This SIP’s
+copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+
+The Stacks network controls total runtime and I/O expenditure during Nakamoto tenures
+via a total tenure budget. Costs are tracked using worst-case estimates of the actual
+runtime or I/O expended to evaluate a given Clarity operation. In Nakamoto rules,
+signers and miners may coordinate to include a `TenureChange` transaction in the miner's
```

Comment:
How does this SIP affect the effectiveness of the coordination between signers and miners? Before there were only two options which are less likely to be disputed.

The SIP should address what happens if coordination fails, or why it is likely to improve the situation.

Reply:
Added in 97719e1